Resources/Private/Language/locallang_db.xlf

```diff
@@ -137,6 +137,12 @@
                 <source>theodia</source>
             </trans-unit>
 
+            <trans-unit id="site.tx_theodia_api_key">
+                <source>API key</source>
+            </trans-unit>
+            <trans-unit id="site.tx_theodia_api_key.description">
+                <source>The API key allowing access to theodia's data.</source>
```

Comment:
If the way to get the API key is to to go to some website, it would be useful to have this info in the description as well.